subworkflows/nf-core/fastq_align_chromap/main.nf

```diff
@@ -0,0 +1,41 @@
+/*
+ * Map reads, sort, index BAM file and run samtools stats, flagstat and idxstats
+ */
+
+include { CHROMAP_CHROMAP         } from '../../../modules/nf-core/chromap/chromap/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow FASTQ_ALIGN_CHROMAP {
+    take:
+    ch_reads           // channel: [ val(meta), [ reads ] ]
+    ch_index           // channel: [ val(meta2, [ index ] ]
+    ch_fasta           // channel: [ val(meta2, [ fasta ] ]
+    ch_barcodes        // channel: [ barcodes ]
+    ch_whitelist       // channel: [ whitelist ]
+    ch_chr_order       // channel: [ chr_order ]
+    ch_pairs_chr_order // channel: [ pairs_chr_order ]
```

Comment:
Maybe mention these are still optoinal?

Reply:
Done in [1aadccf](https://github.com/nf-core/modules/pull/2414/commits/1aadccfb1acf163644a7849643ee8efc5b11ef5e)